README.md

```diff
@@ -4,6 +4,25 @@ This Yocto meta layer contains all the recipes needed to build the Mender client
 
 For instructions on using this meta layer as part of a Yocto build environment, please see [the Mender documentation](https://docs.mender.io/system-updates-yocto-project).
 
+## Maintenance strategy
+
+The repository follows the release-named branch strategy.
+
+### Current branch state
+
+This branch `master` is in **development** maintenance.
```

Comment:
I wouldn't say "this" branch, because we will eventually branch out and this README will end up in other branches.
```suggestion
Branch `master` is in **development** maintenance.
```

Following that reasoning, I don't know what to do with the title "Current branch state" :thinking: Is your idea to modify this README in every branch and keep it up to date forever?

Reply:
This readme addition is actually intended to go to all branches, to name their respective states right away, exactly.

Reply:
Ok.